src/util/dom_style.js

```diff
@@ -1,5 +1,6 @@
 (function (global) {
   var fabric = global.fabric;
+  /* this file needs to go away, cross browser style support is not fabricjs domain.
```

Comment:
i didn't know we had this, but we need to remove it. Is probably for old days.
Safari eventuall doesn't support yet opacity in stile, meaning that opacity in style shouldn't be used imho.

Reply:
I am shocked that opacity isn't supported... WHY?
Well, how does it affect fabric?

Reply:
it doesn't. I think that if you pass a style object to the canvas, we transfer it on the container using this setStyle function.
And if for some weird reason you want your canvas with the opacity style property.
But really is not our problem to fix.